lwt-ppx.opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+name: "lwt"
+version: "dev"
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Mauricio Fernandez <mfp@acm.org>"
+  "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+]
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt/manual/"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+dev-repo: "https://github.com/ocsigen/lwt.git"
+build: [ [ "jbuilder" "build" "-p" name "-j" jobs ] ]
+build-test: [ [ "jbuilder" "runtest" ] ]
+depends: [
+  "ocamlfind" {build & >= "1.5.0"}
+  "jbuilder" { build & >= "1.0+beta9" }
+  "lwt-core"                            # is this really needed, or only sensible?
```

Comment:
I would remove it if it's not entirely necessary. It will give opam an opportunity to parallelize the builds more.